packages/ui/src/components/eventTimelineWidget.js

```diff
@@ -30,30 +31,41 @@ class EventTimelineWidget extends Component {
     }
 
     /**
+     * @name loadEventDetails
+     * @desc Move to event details page
+     * @return Loads the event details page
+     */
+    loadEventDetails = (id) => {
+      console.log(id)
+      History.push(`/eventDetails/${id}`)
+    }
+
+    /**
      * @name renderEvents
      * @desc Iterates through the list of the events and renders the list of events
      * @return Rendered events list {HTML}
      */
     renderEvents = () => {
         return this.props.events.map((event, index) => {
-            return <ListItem 
-            key={index} 
+            return <ListItem
+            key={index}
             primaryText={event.title}
```

Comment:
Replace primary text to <Link to={routepath}>{event.title}</Link>

Refer https://reacttraining.com/react-router/web/example/auth-workflow